configs/ttalps_analysis/ttalps_histogrammer_list.py

```diff
@@ -92,6 +94,12 @@
   "collision_data2018/SingleMuon2018D",
   
   # Signal
+  # "signals/tta_mAlp-0p35GeV_ctau-1e0mm_nEvents-100",
+  # "signals/tta_mAlp-0p35GeV_ctau-1e1mm_nEvents-100",
+  # "signals/tta_mAlp-0p35GeV_ctau-1e2mm_nEvents-100",
+  # "signals/tta_mAlp-0p35GeV_ctau-1e3mm_nEvents-100",
+  # "signals/tta_mAlp-0p35GeV_ctau-1e5mm_nEvents-1000",
+  # "signals/tta_mAlp-0p35GeV_ctau-1e7mm_nEvents-1000",
   # "signals/tta_mAlp-0p35GeV_ctau-1e2mm",
   # "signals/tta_mAlp-0p35GeV_ctau-1e3mm",
   # "signals/tta_mAlp-0p35GeV_ctau-1e5mm",
```

Comment:
I think at some stage we should get rig of this nEvents part of the filename - it doesn't really matter once the sample is generated.